packages/cdc-react/src/components/Checkbox/Checkbox.tsx

```diff
@@ -41,12 +41,14 @@ export const Checkbox = (props: CheckboxProps) => {
         onChange={props.onChange}
       />
       {Icon}
-      <div className="checkbox-content">
-        <span className="label">{props.label}</span>
-        {props.description && (
-          <div className="label-description">{props.description}</div>
-        )}
-      </div>
+      {(props.label || props.description) && (
```

Comment:
Not critical at this point but we will need to require a label for accessibility/508. If we don't want to explicitly show it there are a couple workarounds. In the table example we would probably want to use the file name such that a no- or low- vision user could keyboard navigate through the table and have context on which row they are on.

Reply:
Thank you! So, I updated the styling, added a keyboard keydown event handler for selecting a row from the table, and added the labels to the checkboxes for screenreading.